go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/improbable-eng/grpc-web v0.12.0
 	github.com/logrusorgru/aurora v0.0.0-20200102142835-e9ef32dff381
 	github.com/onflow/cadence v0.14.4
-	github.com/onflow/flow-go v0.15.4
+	github.com/onflow/flow-go v0.15.5-0.20210413143122-9662233e328e // https://github.com/onflow/flow-go/pull/616 TODO before merge replace with tag!
```

Comment:
🔍

Reply:
@turbolent I am ready to merge this, but I cannot use the latest tag of flow-go here because it depends on a private repo:

```
go get github.com/onflow/flow-go@v0.16.2
go get: github.com/onflow/flow-go@v0.16.2 requires
	github.com/onflow/cadence@v0.15.1-patch.1: reading github.com/onflow/cadence/go.mod at revision v0.15.1-patch.1: unknown revision v0.15.1-patch.1
```

I assume `v0.15.1-patch.1` is in `github.com/dapperlabs/cadence-internal` but I don't have access to it.

What do you suggest?